### DISCUSSION TITLE
[Feature Request] more human-readable display of response codes

### DISCUSSION THREAD

**Discussion by bboreham** [grafana/grafana#37245] [General]
**What would you like to be added**:
A message like "Server responded: Too Many Requests", which is the meaning of a 429 response. 

**Why is this needed**:
Right now I see this:
![image](https://user-images.githubusercontent.com/8125524/123404680-feb9b880-d5a0-11eb-9c50-ac401e92e66b.png)

In the console:
![image](https://user-images.githubusercontent.com/8125524/123404896-388abf00-d5a1-11eb-8c7e-8178c3afae9d.png)

I was using Grafana v8.0.3.

**Comment by zuchka** [comment]
Hi @bboreham 

Thanks for submitting this. Sounds like a useful enhancement.  I'm curious: did you notice how the grafana logs handled this error? Thanks

**Comment by bboreham** [comment]
Can you rephrase your question?  What logs?

I posted about as much as I looked at in logs.

**Comment by zuchka** [comment]
sorry @bboreham; maybe a silly question! I was just curious if the browser console logs and the Grafana Server logs printed the same info.

**Comment by dannykopping** [comment]
Adding another data-point here:

When a request is too large for the receiving HTTP server to accept, it returns a [414 URI Too Long](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/414) error. In my case, this was because a Prometheus datasource was set to `GET` instead of `POST`. I received the same error message display as @bboreham has described above, and can confirm that there is no meaningful log produced by Grafana in the JS console.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
